recipes/libgfortran-abi/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set version = "1.5.0" %}
+
+package:
+  name: libgfortran-abi
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+
+requirements:
+  run:
+    - libgfortran 3.0.0
```

Comment:
I think of the ABI as something like an attribute of the libgfortran package.  The only dependency that matters is the libgfortran dependency on the ABI.  Make sense?  If not let's discuss more.  This is the first implementation here outside of discussions, so let's make sure we all understand.